packages/frontend-api/src/Model/Mutation/Customer/User/CustomerUserMutation.php

```diff
@@ -0,0 +1,144 @@
+     */
+    protected $customerUserPasswordFacade;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Customer\User\CustomerUserRefreshTokenChainFacade
+     */
+    protected $customerUserRefreshTokenChainFacade;
+
+    /**
+     * @var \Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface
+     */
+    protected $tokenStorage;
+
+    /**
+     * @var \Shopsys\FrontendApiBundle\Model\Customer\User\CustomerUserUpdateDataFactory
+     */
+    protected $customerUserUpdateDataFactory;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Customer\User\CustomerUserFacade
+     */
+    protected $customerUserFacade;
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Customer\User\FrontendCustomerUserProvider $frontendCustomerUserProvider
+     * @param \Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface $userPasswordEncoder
+     * @param \Shopsys\FrameworkBundle\Model\Customer\User\CustomerUserPasswordFacade $customerUserPasswordFacade
+     * @param \Shopsys\FrameworkBundle\Model\Customer\User\CustomerUserRefreshTokenChainFacade $customerUserRefreshTokenChainFacade
+     * @param \Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface $tokenStorage
+     * @param \Shopsys\FrontendApiBundle\Model\Customer\User\CustomerUserUpdateDataFactory $customerUserUpdateDataFactory
+     * @param \Shopsys\FrameworkBundle\Model\Customer\User\CustomerUserFacade $customerUserFacade
+     */
+    public function __construct(
+        FrontendCustomerUserProvider $frontendCustomerUserProvider,
+        UserPasswordEncoderInterface $userPasswordEncoder,
+        CustomerUserPasswordFacade $customerUserPasswordFacade,
+        CustomerUserRefreshTokenChainFacade $customerUserRefreshTokenChainFacade,
+        TokenStorageInterface $tokenStorage,
+        CustomerUserUpdateDataFactory $customerUserUpdateDataFactory,
+        CustomerUserFacade $customerUserFacade
+    ) {
+        $this->frontendCustomerUserProvider = $frontendCustomerUserProvider;
+        $this->userPasswordEncoder = $userPasswordEncoder;
+        $this->customerUserPasswordFacade = $customerUserPasswordFacade;
+        $this->customerUserRefreshTokenChainFacade = $customerUserRefreshTokenChainFacade;
+        $this->tokenStorage = $tokenStorage;
+        $this->customerUserUpdateDataFactory = $customerUserUpdateDataFactory;
+        $this->customerUserFacade = $customerUserFacade;
+    }
+
+    /**
+     * @param \Overblog\GraphQLBundle\Definition\Argument $argument
+     */
+    public function changePassword(Argument $argument)
+    {
+        $input = $argument['input'];
+
+        try {
+            $customerUser = $this->frontendCustomerUserProvider->loadUserByUsername($input['email']);
+        } catch (UsernameNotFoundException $e) {
+            throw new UserError('User does not exists or provided password is not valid.');
```

Comment:
This is intentioned to not allow any 3rd party check for existing emails registrations. Same logic is applied in LoginController. But I will update the message to "This account doesn't exist or password is incorrect" in order to be consistent with LoginController.

Reply:
This is common excuse but in fact you can check for existing email via registration. Here it makes just bad user experience.
e.g. I am using 2 or more emails and not sure which one I am registered with. I need to try all my emails with all passwords combined because I do not know what actually is wrong.

Reply:
I do not think that you can use registration for that as you would create multiple users and this could lead to some investigation.

I actually prefer this one

Reply:
Actually you can. When you try to log in with email and the response i `Email is already taken` you have an information the email is in the system.

But I do not want to blok this PR just because of error message. Please discuss it while BV or we can discuss it more globaly some day.

Reply:
I agree with @TomasLudvik, lets keep the error message more generic and consistent....
"This account doesn't exist or password is incorrect"